### PR TITLE
fix: updated the ember-cli version to the latest release

### LINKED ISSUE #73: Update ember-cli version

### PR DESCRIPTION
**Pull request by Brocco**
Fixes #73 by updating the ember-cli version to the latest release

https://github.com/ember-cli/ember-cli/releases/tag/v1.13.13

Minor build issue fixed via helper update to mock-ui.js (pulled directly from ember's helper class)

### CODE REVIEW

**Comment by Brocco** [comment]
The build failure will go away with the pull request for issue #75

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "broccoli-merge-trees": "^0.2.1",
     "broccoli-writer": "^0.1.1",
     "chalk": "^1.1.0",
-    "ember-cli": "ember-cli/ember-cli#ae5e72614e40f5e6dd57463e526c65da899fde07",
+    "ember-cli": "ember-cli/ember-cli#ef6b848b38d85037f917dcc14a6735df59a94ea7",
```
Comment (cironunes) [inline package.json]: can we use the version number instead of the commit hash?

**Comment by Brocco** [comment]
I'm fine with doing that, we can even pull in the latest version via npm directly instead of via github?

so either:

 - github via version/tag
 - npm w/ exact version

**Comment by cironunes** [comment]
Awesome! Please amend the commit changing to the exact version and it will be good to merge.

**Comment by Brocco** [comment]
This also now includes the change that fixes #75 (unit test failure)

**Comment by cironunes** [comment]
@Brocco the best would be having them in two different commits. Can you just remove the change and fix the commit message?

**Comment by IgorMinar** [comment]
lots of test failures. @Brocco can you please investigate?

**Comment by IgorMinar** [comment]
ping

**Comment by Brocco** [comment]
Node v5 checks were failing had to update some broccoli package references, all is good now

**Comment by cironunes** [comment]
ci green, merged. Thanks @Brocco

**Pull request merged**